db/sample_data.rb

```diff
@@ -36,35 +36,34 @@ def add_sample_data
-  application_letters[:applicant_gongakrobatik_past_deadline] = application_letter_applicant_gongakrobatik(users[:tobi], events[:past_deadline_event])
-  application_letters[:applicant_gongakrobatik_accepcted] = application_letter_applicant_gongakrobatik_accepted(users[:lisa], events[:past_deadline_event])
-  application_letters[:applicant_gongakrobatik_rejected] = application_letter_applicant_gongakrobatik_rejected(users[:applicant], events[:past_deadline_event])
-  application_letters[:applicant_gongakrobatik_max] = application_letter_applicant_gongakrobatik_accepted(users[:max], events[:past_deadline_event])
-  application_letters[:applicant_gongakrobatik_karl] = application_letter_applicant_gongakrobatik_accepted(users[:pupil], events[:past_deadline_event])
+  application_letters[:applicant_gongakrobatik] = application_letter_1(users[:applicant], events[:gongakrobatik])
+  application_letters[:applicant_gongakrobatik_past_deadline] = application_letter_1(users[:keven], events[:past_deadline_event])
+  application_letters[:applicant_gongakrobatik_accepcted] = application_letter_2(users[:lisa], events[:past_deadline_event])
+  application_letters[:applicant_gongakrobatik_rejected] = application_letter_3(users[:applicant], events[:past_deadline_event])
+  application_letters[:applicant_gongakrobatik_max] = application_letter_4(users[:ralf], events[:past_deadline_event])
+  application_letters[:applicant_gongakrobatik_karl] = application_letter_5(users[:pupil], events[:past_deadline_event])
   application_letters[:applicant_programmierkurs_lisa] = application_letter_applicant_programmierkurs_1(users[:lisa], events[:programmierkurs])
-  application_letters[:applicant_programmierkurs_max] = application_letter_applicant_programmierkurs_2(users[:max], events[:programmierkurs])
-  application_letters[:applicant_programmierkurs_tobi] = application_letter_applicant_programmierkurs_3(users[:tobi], events[:programmierkurs])
+  application_letters[:applicant_programmierkurs_max] = application_letter_applicant_programmierkurs_2(users[:ralf], events[:programmierkurs])
+  application_letters[:applicant_programmierkurs_tobi] = application_letter_applicant_programmierkurs_3(users[:keven], events[:programmierkurs])
 
   application_letters[:applicant_mintcamp_lisa] = application_letter_applicant_programmierkurs_2(users[:lisa], events[:mintcamp])
-  application_letters[:applicant_mintcamp_max] = application_letter_applicant_programmierkurs_1(users[:max], events[:mintcamp])
-  application_letters[:applicant_mintcamp_tobi] = application_letter_applicant_programmierkurs_3(users[:tobi], events[:mintcamp])
+  application_letters[:applicant_mintcamp_max] = application_letter_applicant_programmierkurs_1(users[:ralf], events[:mintcamp])
+  application_letters[:applicant_mintcamp_tobi] = application_letter_applicant_programmierkurs_3(users[:keven], events[:mintcamp])
```

Comment:
If you really want to fix this, please do this on your own. We do not have time for this perfectionism